cmd/telegraf/telegraf_posix.go

```diff
@@ -12,3 +17,13 @@ func (t *Telegraf) Run() error {
 func cliFlags() []cli.Flag {
 	return []cli.Flag{}
 }
+
+func getLockedMemoryLimit() uint64 {
+	const RLIMIT_MEMLOCK = 8
+
+	var limit syscall.Rlimit
+	if err := syscall.Getrlimit(RLIMIT_MEMLOCK, &limit); err != nil {
+		panic(fmt.Errorf("Cannot get limit for locked memory: %w", err))
```

Comment:
hmm panic on start up in a function that everyone is going to start running scares me. Should this be a warning as well?

Reply:
Hmmm what do you suggest? Warn and then? Return zero?

Reply:
I like returning 0 as it will print the warning in the logs and it would be clear to us that looking up the limit failed, but the user can keep going.